ruby/README.md

```diff
@@ -0,0 +1,57 @@
+# ThreatExchange
+
+A ruby library to interface with Facebooks ThreatExchange API. This is still very much in development so feel free to contribute :)
+
+## Installation
+Install required dependencies 
+
+    bundle install
+
+Install this gem locally
+
+    rake install
+
+Add this to your application
+
+    require 'ThreatExchange'
+
+## Usage
+
+The ThreatExchange library provides a base set of abstractions for the ThreatExchange API. You instanciate a new ThreatExchange::Client and provide it with your access token.
+
+```ruby
+
+access_token = 'abc123'
+TE = ThreatExchange::Client.new(access_token)
```

Comment:
Can you please change this example to have the format of an actual access token:

    access_token = '56356356|367665625651'